installing/installing_sno/install-sno-installing-sno.adoc

```diff
@@ -6,9 +6,8 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install {sno} using the web-based Assisted Installer and a discovery ISO that you generate using the Assisted Installer. You can also install {sno} by using `coreos-installer` to generate the installation ISO.
-
 ifndef::openshift-origin[]
+You can install {sno} using the web-based Assisted Installer and a discovery ISO that you generate using the Assisted Installer. You can also install {sno} by using `coreos-installer` to generate the installation ISO.
```

Comment:
This bit would need "how to setup Assisted Installer" section for OKD, as users can't use one from console.redhat.com.

I think a simple ref to https://github.com/openshift/assisted-service/tree/master/deploy/podman#okd-configuration would be good for a start

Reply:
Thanks, I wasn't aware of that document.
I just updated the file and pushed the changes.